funcx/endpoint/endpoint.py

```diff
@@ -260,46 +254,47 @@ def start_endpoint(args, global_config=None):
                                        # lockfile.FileLock(
                                        pidfile=daemon.pidfile.PIDLockFile(os.path.join(endpoint_dir,
                                                                                        'daemon.pid')),
-                                       stdout=stdout,
+                                       stdout=sys.stdout,
```

Comment:
I suspect this is a typo. We want these streams to go to log files because they would otherwise be inaccessible.

Reply:
yes, this was a mistake.  I had been trying to see if I could get the daemon context to give the uuid back to the parent process in a simple-ish way because Ryan remarked that sometimes the command could fail, but since it printed the uuid, it wasn't immediately clear to the user.  I've cleaned that up